Angular/src/app/discussion-list/discussion-list.component.html

```diff
@@ -1,13 +1,26 @@
 <h3>List of Discussions</h3>
-<form action="/action_page.php">
-    <input matInput (keyup)="applyFilter($event.target.value)"name="search">
-   
-</form>
+<div class="input-group">
```

Comment:
All of the angular will be moved out next sprint. Leaving things commented out incase angular frontend repo needs them.

Reply:
ok.